csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -539,23 +538,40 @@ void HopperMultipleMatmulScheduler::scheduleEpilogue() {
         transformLikeMmaOutput(tv, /*is_mma_result=*/false);
       }
 
-      auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-          dc->getLoopDomain());
-      dc->setLoopDomain(s.as<IterDomain*>());
-      dc->setAllocationDomain(s.as<IterDomain*>(), true);
-
-      scheduler_utils::BoundedDirectionalTransformPropagator::backward(
-          dc,
-          -1,
-          propagate_to,
-          scheduler_utils::BoundedDirectionalTransformPropagator::Options()
-              .propagateParallelType());
+      // Should not propagate if the dc is a mma output.
```

Comment:
The problem is not even in the propagation, but the fact that we do not want to schedule those `dc` tensors as if they are unscheduled when they have already been scheduled as mma results.
```suggestion
      // When dc is also an mma_result, it will have already been scheduled by scheduleMmaResults.
      // In that case, we need to avoid scheduling it a second time here.
```